content/api/commands/intercept.md

```diff
@@ -2,156 +2,112 @@
-- and much more - `cy.intercept()` gives full access to all HTTP requests at all stages.
+<Alert type="warning">
 
-## Comparison to `cy.route()`
+`cy.route` was replaced by `cy.intercept` in Version 6.0.0. See [Comparison to `cy.route`](#Comparison-to-cy-route).
 
-Unlike [cy.route()](/api/commands/route), `cy.intercept()`:
+</Alert>
 
-- can intercept all types of network requests including Fetch API, page loads, XMLHttpRequests, resource loads, etc.
-- does not require calling [cy.server()](/api/commands/server) before use - in fact, `cy.server()` does not influence `cy.intercept()` at all.
-- does not have method set to `GET` by default, but intercepts `*` methods.
+## Syntax and Usage
 
-## Usage
+#### Without `routeHandler` (Spying Only)
 
-```ts
-cy.intercept(url, routeHandler?)
-cy.intercept(method, url, routeHandler?)
-cy.intercept(routeMatcher, routeHandler?)
+```js
+cy.intercept(url)
+cy.intercept(method, url)
+cy.intercept(routeMatcher)
+```
+
+#### With `routeHandler` (Request/Response Modification and Spying)
+
+```js
+cy.intercept(url, routeHandler)
+cy.intercept(method, url, routeHandler)
+cy.intercept(routeMatcher, routeHandler)
 cy.intercept(url, routeMatcher, routeHandler)
 ```
 
-**Note:** all intercepts are automatically cleared before every test.
+<Alert type="warning">
 
-### Arguments
+All intercepts are automatically cleared before every test.
 
-#### **<Icon name="angle-right"></Icon> url** **_(`string | RegExp`)_**
+</Alert>
 
-Specify the URL to match. See the examples for [Matching URL](#Matching-URL) to see how URLs are matched.
+**<Icon name="check-circle" color="green"></Icon> Correct Usage**
 
-```ts
-cy.intercept('http://example.com/widgets')
-cy.intercept('http://example.com/widgets', { fixture: 'widgets.json' })
+```js
+cy.intercept('/users/**')
+cy.intercept('GET', '/users*')
+cy.intercept({ method: 'get', url: '/users*', hostname: 'localhost' })
+cy.intercept({ method: 'POST', url: '/users*' }, { success: true })
+cy.intercept('/users*', { hostname: 'localhost' }, (req) => {
+  /* do something with request */
+})
 ```
 
-#### **<Icon name="angle-right"></Icon> method** **_(`string`)_**
-
-Specify the HTTP method to match on.
+**<Icon name="exclamation-triangle" color="red"></Icon> Incorrect Usage**
 
-```ts
-cy.intercept('POST', 'http://example.com/widgets', {
-  statusCode: 200,
-  body: 'it worked!',
-})
+```js
+cy.intercept('GET')
+// `method` argument must be passed in with `url`
+cy.intercept({ success: true })
+// `routeHandler` cannot be passed in by itself
+cy.intercept('/users*', { method: 'GET' })
+// `url` with `routeMatcher` requires `routeHandler`
+cy.intercept('GET', '/users*', { hostname: 'localhost' })
+// `method` can't be passed in with `routeMatcher`
+cy.intercept('GET', '/users*', { hostname: 'localhost' }, { success: true })
+// this command only accepts up to three arguments
```

Comment:
I'll have to think about an alternate way of presenting this. In the meantime, since this was @amirrustam's suggestion,  I'd like to give him an opportunity to chime in.

Reply:
I can remove this and punt an issue to the next iteration to figure it out.

Reply:
See DX-183